app/(root)/case-studies/page.tsx

```diff
@@ -1,7 +1,5 @@
 import CaseStudies from '@/components/case-studies/CaseStudies';
 
-const CaseStudy = () => {
-  return <CaseStudies />;
-};
+const Page = () => <CaseStudies />;
```

Comment:
What is your reasoning for changing the name back to Page?

Reply:
![image](https://github.com/AlexDjangoX/portfolio-alexander/assets/79525229/c7b45557-41f9-4d7a-9ee7-dd2adf555e96)

It was changed to this while refactoring using Visual Studio. 

I have called it CaseStudy. More descriptive.